roles/openshift_hosted_facts/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Set hosted facts
+  openshift_facts:
+    role: hosted
+    openshift_env: "{{ item | oo_openshift_env }}"
+  no_log: true
+  with_items:
+  - "{{ hostvars[inventory_hostname] }}"
+  - "{{ hostvars }}"
```

Comment:
Is it possible to do a union of hostvars and hostvars[inventory_hostname] instead of the loop?